crawler/crawler/cloneRepository.go

```diff
@@ -33,17 +33,12 @@ func CloneRepository(domain Domain, hostname, name, gitURL, gitBranch, index str
 		if err != nil {
 			return errors.New(fmt.Sprintf("cannot git pull the repository: %s: %s", err.Error(), out))
 		}
-		// Command is: git reset --hard origin/<branch_name>
-		out, err = exec.Command("git", "-C", path, "reset", "--hard", "origin/"+gitBranch).CombinedOutput() // nolint: gas
-		if err != nil {
-			return errors.New(fmt.Sprintf("cannot git pull the repository: %s: %s", err.Error(), out))
-		}
 		return err
 	}
 
 	// Clone the repository using the external command "git".
-	// Command is: git clone -b <branch> <remote_repo>
-	out, err := exec.Command("git", "clone", "-b", gitBranch, gitURL, path).CombinedOutput() // nolint: gas
+	// Command is: git clone --filter=blob:none --mirror -b <branch> <remote_repo>
```

Comment:
I'd just remove those comments, they've never added any useful info

Reply:
Removed in 8fc0f72a27a5c35245bdb2b13dffa82b676cb77e.